src/components/ActivityCard.js

```diff
@@ -20,16 +22,9 @@ export default function Card({
         <LinkStyling to={`/${nameOfSelectedFriend}/${nameOfSelectedActivity}`}>
           <strong>{activity}</strong>
         </LinkStyling>
-        <Button
-          background="transparent"
-          justifySelf="end"
-          padding="10px"
-          width="auto"
-          height="auto"
-          onClick={() => setShow(true)}
-        >
+        <DeleteButton onClick={() => setShow(true)}>
           <img src={deleteicon} alt="delete" />
```

Comment:
depends how you want to style your icons...you could go inline svg here or even create a icon component! advantage is that you have more control of styling...

Reply:
Thats a good idea, I also checked how you did it 😊
I think i will keep it for now like this, as i am using them straight away as they are and actually not really style them..